docs/getting-started/gs-faqs.md

```diff
@@ -185,6 +194,12 @@ When a user opts out of UID2-based targeted advertising through the [Transparenc
 
 If a user opts out through your website, you should follow your internal procedures for handling the opt-out. For example, you might choose not to generate a UID2 for that user.
 
+#### Does the same DII always result in the same raw UID2?
+xxx
+In general yes, the process of generating a raw UID2 from DII is the same, and results in the same value, no matter who sent the request. If two UID2 participants were to send the same email address to the [POST /identity/map](../endpoints/post-identity-map.md) endpoint at the same time, they would both get the same raw UID2 in response.
+
+However, there is a variable factor, which is the [salt](../ref-info/glossary-uid.md#gl-salt) value that's used in generating the raw UID2. The salt values are rotated periodically. If the salt value changes between one request and another, those two requests result in two different raw UID2, even when the DII is the same.
```

Comment:
Shall we add a link to the integration guide where salt bucket is discussed as "For further info"?
https://unifiedid.com/docs/guides/advertiser-dataprovider-guide#3-monitor-for-salt-bucket-rotations-related-to-your-stored-raw-uid2s

Reply:
@sunnywu Great suggestion. Done. Thx.